auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -410,13 +412,23 @@ public SignInIntentBuilder setProviders(@NonNull String... providers) {
         /**
          * Enables or disables the use of Smart Lock for Passwords in the sign in flow.
          * 
-         * <p>SmartLock is enabled by default
+         * <p>SmartLock is enabled by default.
          */
         public SignInIntentBuilder setIsSmartLockEnabled(boolean enabled) {
             mIsSmartLockEnabled = enabled;
             return this;
         }
 
+        /**
+         * Links the current user to one created in the sign in flow.
+         *
+         * <p>Linking is disabled by default.
+         */
+        public SignInIntentBuilder linkWithCurrentUser(boolean shouldLinkUser) {
```

Comment:
nit: name this `setShouldLinkUser` or something more consistent with `setIsSmartLockEnabled` and the others.